protobufs/livekit_models.proto

```diff
@@ -266,7 +266,8 @@ message SipDTMF {
 }
 
 message Transcription {
-  string participant_identity = 2;
+  // Participant that got its speech transcribed
+  string transcribed_participant_identity = 2;
```

Comment:
seems redundant to me? everything in a Transcription should be scoped to the transcription.

Reply:
That's a part of https://github.com/livekit/protocol/pull/706 probably. It was confusing to see two `participant_identities` in data packets in SDK.

Reply:
I can see that.. would comments in the protobuf help to clear that up? Just looking at a `Transcription` object (without context on what is using the message), it seems to break some naming patterns to have `transcribed_` again

Reply:
Maybe `source_` then?

Reply:
that sgtm!